sites/partners/src/components/listings/ListingFormActions.tsx

```diff
@@ -445,7 +447,8 @@ const ListingFormActions = ({
         if (lotteryResults) {
           elements.push(editPostedResultsButton(lotteryResults))
```

Comment:
I wonder if we should also comment out the edit button in case a listing has results, which I don't think is true, wdyt? Could go either way

Reply:
Are there already lottery results posted on Doorway though?

Reply:
I don't think so, but if there were, I think the edit button would also allow them to add results. I can try to check!

Reply:
Update it's actually gonna be kinda hard to check, while it's unlikely would you be okay commenting the edit results button out as well as part of this just in case?